controllers/controller/devworkspacerouting/solvers/common_test.go

```diff
@@ -0,0 +1,125 @@
+package solvers
+
+import (
+	"testing"
+
+	"github.com/devfile/devworkspace-operator/apis/controller/v1alpha1"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetRouteForEndpointAnnotations(t *testing.T) {
+	tests := []struct {
+		name string
+
+		routingSuffix string
+		endpoint      v1alpha1.Endpoint
+		meta          DevWorkspaceMetadata
+		annotations   map[string]string
+
+		expectedAnnotationsKeys []string
```

Comment:
Might almost feel redundant, but it might be nicer to also compare the resulting annotation values (in addition to the annotation keys) with the expected key/values. So this could just be `expectedAnnotations map[string]string`

Reply:
Yes. I blow my mind with the generated annotation for Endpoint name, but this value is controlled by the test runner. So, a simple Equal should be enough.